### PR TITLE
Code Review Project: Review One of My Old Projects‏

### PR DESCRIPTION
**Pull request by Reem-lab**
## Code review needed ♻️ 
- ➕ Add all project files to the repo 
    - index.html file 
    - style.css file 
    - all local linters files
    -  .gitignore  file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3,18 +3,47 @@
...
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="description"
-  content="full description of the app">
-  <meta name="robot" content="index,follow">
-  <link rel="stylesheet" href="styles/reset.css">
-  <link rel="stylesheet" href="styles/debug.css">
-  <title>Document | Description</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Document</title>
```
Comment (Reem-lab) [inline index.html:9]: - [ ] I believe it would be better if you change your title to a more descriptive one. ♻️ 🤔

File: index.html [hunk 2]
```diff
@@ -3,18 +3,47 @@
...
+  <main id="main">
+    <h1 id="title">Dr. Norman Borlaug</h1>
+    <figure id="img-div">
+      <img id="image" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg"
+        alt="the person">
+      <figcaption id="img-caption">increase wheat yields - part of his life-long war on hunger.</figcaption>
+    </figure>
+
+    <div id="tribute-info">
+      <p> Here's a time line of Dr. Borlaug's life:</p>
```
Comment (Reem-lab) [inline index.html:22]: - [ ] I see it should be a header h2 rather than paragraph  👍

File: index.html [hunk 3]
```diff
@@ -3,18 +3,47 @@
...
+        </li>
+        <li>
+          1933 - Leaves his family's farm to attend the University of Minnesota, thanks to a Depression era program
+          known as the "National Youth Administration"
+        </li>
+        <li>
+          1935 - Has to stop school and save up more money. Works in the Civilian Conservation Corps, helping
+          starving Americans. "I saw how food changed them", he said. "All of this left scars on me."
+        </li>
+      </ul>
```
Comment (Reem-lab) [inline index.html:35]: - [ ] I think here that would be more semantic to separate date from the description using `dt` and `dd` tags and enclose the whole section in `dl` tag 💯

File: index.html [hunk 4]
```diff
@@ -3,18 +3,47 @@
...
+
+    </div>
+
+    <p>
```
Comment (Reem-lab) [inline index.html:39]: - [ ] it would be better to add a class for the `p` tag and be more descriptive. 👍

File: style.css [hunk 5]
```diff
@@ -0,0 +1,48 @@
+*,
+body {
+  margin: 0;
+  padding: 0;
+  font-family: sans-serif;
```
Comment (Reem-lab) [inline style.css:5]: - [ ] it is better to add the `font-family` property to the body element and all children elements are going to inherit that property by default. ♻️ ‏

File: style.css [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+}
+
+body {
+  width: 100%;
+  max-width: 1000px;
+  margin: 0 auto;
+}
+
+#title {
+  text-align: center;
```
Comment (Reem-lab) [inline style.css:15]: - [ ] Good job so far 👍  however you can use a separate class for `text-align:  center;` to make your code more DRY ❗ 😄

**Review by Reem-lab — COMMENTED** [review]
## Hi @Reem-lab , 👋 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the prev‏

**Review by moise10r — APPROVED** [review]
Hi @Reem-lab , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
approve1


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**